test/brod_group_coordinator_SUITE.erl

```diff
@@ -131,6 +134,20 @@ t_acks_during_revoke(Config) when is_list(Config) ->
 
   ok.
 
+t_update_topics_triggers_rebalance(Config) when is_list(Config) ->
+    {ok, GroupCoordinatorPid} =
+    brod_group_coordinator:start_link(?CLIENT_ID, ?GROUP, [?TOPIC],
+                                      _Config = [], ?MODULE, {self(), 1}),
+  ?assert_receive({assignments_revoked, 1}, ok),
+  GroupCoordinatorPid ! continue,
+  GenerationId1 = ?assert_receive({assignments_received, 1, GId1, _}, GId1),
+
+  brod_group_coordinator:update_topics(GroupCoordinatorPid, [?TOPIC1]),
+  ?assert_receive({assignments_revoked, 1}, ok),
+  GroupCoordinatorPid ! continue,
+  GenerationId2 = ?assert_receive({assignments_received, 1, GId2, _}, GId2),
```

Comment:
Check that the new assignments include both `?TOPIC` and `?TOPIC1`?

Reply:
We replace the whole list (in line with how the java consumer does it), so i suppose i should check there is `?TOPIC1`.

Reply:
Ok, fair enough. Please reflect this in the test.